group_vars/matrix_servers

```diff
@@ -2602,6 +2605,9 @@ matrix_dendrite_systemd_wanted_services_list: |
 #
 ######################################################################
 
+matrix_dendrite_container_extra_arguments: "{{ matrix_homeserver_container_extra_arguments }}"
+matrix_dendrite_app_service_config_files: "{{ matrix_homeserver_container_app_service_config_files }}"
+
```

Comment:
These should be a few lines above, so that they are within the `matrix-dendrite` section.

Reply:
Fixed